src/redprl/categorical_judgment.sml

```diff
@@ -1,42 +1,70 @@
 structure RedPrlCategoricalJudgment :
 sig
   datatype ('sym, 'a) redprl_jdg =
-     EQ of ('a * 'a) * 'a
-   | TRUE of 'a
-   | EQ_TYPE of 'a * 'a
-   | SYNTH of 'a
+    (* `EQ_TYPE ((a, b), k)`:
+     *   `a` and `b` are equal types, taking into account the additional structures specified by `k`.
+     *   (That is, `EQ_TYPE ((a, b), KAN)` means they are equally Kan, not just having the same PER.)
```

Comment:
We should be a bit more cautious in this sentence, since the meaning of type equality in CHiTT is never that the types have the same PER (CHiTT has an intensional type equality). Maybe we should say, "...means that they are equally Kan, in addition to being equal pretypes".